ckan_meta_tester/__init__.py

```diff
@@ -10,10 +12,15 @@ def test_metadata() -> None:
     logging.getLogger('').setLevel(
         environ.get('INPUT_LOG_LEVEL', 'info').upper())
 
+    # Get PR body text
+    pr_url = environ.get('INPUT_PULL_REQUEST_URL')
+    if pr_url:
+        pr_body = requests.get(pr_url).json().get('body', '')
```

Comment:
We should probably use the token to access the API.

https://docs.github.com/en/actions/reference/authentication-in-a-workflow

Reply:
Good point, gonna also set the `Accept:` header while I'm on it.

Reply:
Would this introduce a security risk, though? A third party could set up our Action and pass their own web site as the PR URL, then use that to capture the token. These are just temporary garbage tokens that GitHub generates on the fly, right?

Reply:
> Before each job begins, GitHub fetches an installation access token for the job. The token expires when the job is finished.

If they're quick, they could abuse the token while it's running.
Might have to parse the URL and check the host after all :/

Reply:
This should make it harder to abuse, hoping there's not some route under `https://api.github.com/repos/` with which you can do any shenanigans.